test-common/src/main/java/org/apache/kafka/common/test/TestKitNodes.java

```diff
@@ -96,6 +100,16 @@ public Builder setPerServerProperties(Map<Integer, Map<String, String>> perServe
             return this;
         }
 
+        public Builder setListenerName(ListenerName listenerName) {
```

Comment:
Although we don't support setting a controller at the moment, it would still be better to rename it to `setBrokerListenerName`.